framework/src/multiapps/MultiApp.C

```diff
@@ -918,7 +922,9 @@ MultiApp::globalAppToLocal(unsigned int global_app)
   if (global_app >= _first_local_app && global_app <= _first_local_app + (_my_num_apps - 1))
     return global_app - _first_local_app;
 
-  _console << _first_local_app << " " << global_app << '\n';
+  _console << "processor " << processor_id() << ": " << _first_local_app << "-"
+           << _first_local_app + (_my_num_apps - 1) << " " << global_app << '\n';
+  // _console << _first_local_app << " " << global_app << '\n';
```

Comment:
```suggestion
```

Reply:
I modified this a bit to make it more informative.